README.md

```diff
@@ -87,11 +88,12 @@ sudo docker run --name "postgis" -p 25432:5432 -d -t kartoza/postgis
 ## Environment variables
 
 You can also use the following environment variables to pass a 
-user name, password and/or default database name.
+user name, password and/or default database name(or multiple databases coma separated).
 
 * -e POSTGRES_USER=<PGUSER> 
 * -e POSTGRES_PASS=<PGPASSWORD>
 * -e POSTGRES_DBNAME=<PGDBNAME>
+* -e POSTGRES_MULTIPLE_EXTENSIONS=<postgis> # You can pass as many variables as you need.
```

Comment:
shouldn't this be postgis not <postgis>

Reply:
@gubuntu I will fix it. The default installs these ones
`- POSTGRES_MULTIPLE_EXTENSIONS=postgis,hstore,postgis_topology`

So a user really doesn't need to pass these unless he needs to install something like `postgis_tiger_geocoder`